src/api/test/helpers/db.coffee

```diff
@@ -7,20 +7,29 @@ bcrypt = require 'bcryptjs'
 @fixtures = fixtures = require '../../../test/helpers/fixtures'
 @db = require '../../lib/db'
 
-@fabricate = (collection, data, callback) =>
+@fabricate = (collectionName, data, callback) =>
+  collection = @db.collection(collectionName)
   if _.isArray(data)
-    data = (fixturize collection, obj for obj in data)
+    data = (fixturize collectionName, obj for obj in data)
+    return collection.insertMany data, (err, res) ->
+      collection.find({}).toArray(callback)
+  if collectionName == 'sessions'
```

Comment:
The `mongojs` implementation of `insert` seemed to accept a single document and an array of documents. This is not the case for the native `mongojs` driver, which needs us to be more specific and use `insertOne` or `insertMany`. The test setup for `sessions` seems to be the only one that passes in an array of documents as the argument.